### ISSUE TITLE
Achievements are being reset every day

### ISSUE THREAD

**Issue by crashkonijn** [playgameservices/play-games-plugin-for-unity#1276]
Hi All,

We recently soft-launched our game and we got a bug from many users that their achievements are being reset every day or even when they restart the game. We did notice this behaviour before, but I assumed this had to do with the fact that everyone that was playing until then was signed in as a Beta tester.

Anyway, We've been unable to find any source on the internet that has the same problem so that's why I'm opening this here.

Just to make sure, everyone does get the achievements and the notification so no error's or anything. They just get removed or reset again.

Hopefully you guys can help us out.

Kind regards,

Peter

**Comment by crashkonijn** [comment]
Update, I just had a look at our developers console and somehow 70% of all calls that are being made to google fail with a 4xx response. I thought this information might be usefull.

Anyway, this still doesn't explain why we do still get the achievents at first. (the function call itself also returns success).

We're using unity 5.3.x with GPGS v0.9.32 btw.
![ss 2016-07-10 at 04 49 01](https://cloud.githubusercontent.com/assets/5364802/16714381/7e60a46e-46c1-11e6-8c47-3c1e7f0c1897.png)
![ss 2016-07-10 at 04 48 35](https://cloud.githubusercontent.com/assets/5364802/16714380/7e607840-46c1-11e6-95ea-aa2721f08181.png)

**Comment by crashkonijn** [comment]
Alright we've found the answer: we had 2 credentials and apparently used the wrong one in our APK. We switched around the Signing-certificate fingerprint so that the fingerprint in the APK and in play services match and now it's working.

The strange thing is that everything seemed to work and we've never gotten any feedback from google that something was wrong...

Thanks for your attention to this problem!
